content/en/docs/faq/_index.md

```diff
@@ -10,609 +10,9 @@ aliases: [
 # Frequently Asked Questions
 
 > What are the key differences between Helm 2 and Helm 3?  
```

Comment:
You probably want to remove this line - see how it appears: https://deploy-preview-1104--helm-merge.netlify.app/docs/faq/